src/abinitio.py

```diff
@@ -799,6 +799,7 @@ def callM(mvals_):
                                           col(M_all_print[:,0])-col(Q_all_print[:,0]),
                                           col(self.boltz_wts)))
             np.savetxt("EnergyCompare.txt", EnergyComparison, header="%11s  %12s  %12s  %12s" % ("QMEnergy", "MMEnergy", "Delta(MM-QM)", "Weight"), fmt="% 12.6e")
+            plot_mm_vs_qm(M_all_print[:,0], Q_all_print[:,0], title='Abinitio '+self.name)
```

Comment:
This is called under the condition `if self.writelevel > 0`, so the user can use the keyword `writelevel 1` in the input file to enable it. The performance impact is negligible on my experience.